packages/plugins/users-permissions/admin/src/pages/Roles/ListPage/index.js

```diff
@@ -24,14 +73,19 @@ const RoleListPage = () => {
     defaultMessage: 'Roles',
   });
 
+  const handleClickEdit = id => {
+    push(`/settings/${pluginId}/roles/${id}`);
+  };
+
   return (
     <Layout>
       <SettingsPageTitle name={pageTitle} />
       <Main
+        aria-busy={isLoading}
         labelledBy={formatMessage({
```

Comment:
This one should be the id of the "H1" element, in your case, I think `labelledBy` should be `roles` since on line https://github.com/strapi/strapi/pull/10901/files#diff-a592156f8df4bd83072b0f95b54e635d92de430c19d5c63effc27c376b74bbd1R92 the H1 id is "roles" 😊